src/features/query-bar/__tests__/validator-test.js

```diff
@@ -26,3 +26,33 @@ import { isValidValue } from '../validator';
         expect(actual).toEqual(expected);
     });
 });
+
+describe('isFloat()', () => {
+    test.each`
+        value     | expected | should
+        ${'1.0.'} | ${false} | ${'Should return false if the value is 1.0.'}
+        ${'1.0'}  | ${true}  | ${'Should return true if the value is 1.0'}
+        ${'1'}    | ${true}  | ${'Should return true if the value is 1'}
+        ${'1.a'}  | ${false} | ${'Should return false if the value is 1.a'}
+        ${'a'}    | ${false} | ${'Should return false if the value is a'}
+        ${'0'}    | ${true}  | ${'Should return false if the value is 0'}
+    `('$should', ({ value, expected }) => {
+        const result = isFloat(value);
+        expect(result).toBe(expected);
+    });
+});
+
+describe('isInt()', () => {
+    test.each`
+        value     | expected | should
+        ${'1.0.'} | ${false} | ${'Should return false if the value is 1.0.'}
+        ${'1.0'}  | ${false} | ${'Should return false if the value is 1.0'}
```

Comment:
I'm not sure if this should be false.
in browser's native `isInteger`, 1.0 counts as an integer

Reply:
Good catch. It should be true; I will be using lodash's isInteger to make that test case pass.